providers/os/resources/os.lr.manifest.yaml

```diff
@@ -131,6 +131,10 @@ resources:
     refs:
     - title: What is a container?
       url: https://docs.docker.com/guides/walkthroughs/what-is-a-container/
+  docker.exposes:
```

Comment:
@slntopp should this be included?

Reply:
good catch, was testing smth, cleaned it up